src/dlgTriggerEditor.cpp

```diff
@@ -6685,6 +6708,13 @@ void dlgTriggerEditor::slot_show_timers()
         mpSourceEditorArea->show();
         slot_timer_selected(treeWidget_timers->currentItem());
     }
+    if (mTimerEditorSplitterState.length() > 0) {
```

Comment:
If I can suggest improved readability this way here & down below:

```suggestion
    if (!mTimerEditorSplitterState.isEmpty()) {
```